modules/lvms-download-log-files-and-diagnostics.adoc

```diff
@@ -8,7 +8,7 @@
 
 When {lvms} is unable to automatically resolve a problem, use the must-gather tool to collect the log files and diagnostic information so that you or the Red Hat Support can review the problem and determine a solution.
 
-* Run the must-gather command from the client connected to {lvms} cluster by running the following command:
+* Run the must-gather command from the client connected to {lvms} cluster:
```

Comment:
```suggestion
* Run the `must-gather` command from the client connected to {lvms} cluster:
```

[Example](https://docs.openshift.com/container-platform/4.15/nodes/cma/nodes-cma-autoscaling-custom-debugging.html) of highlighting in other docs.

Reply:
Updated as per the suggestion.